docs/user-guide.md

```diff
@@ -38,7 +38,20 @@ This guide provides detailed instructions on how to configure your Liberty proje
 
 ### Settings
 
-To ensure that the Liberty Config Language Server starts properly, define the `xml.java.home` property in your VS Code settings or set the system `JAVA_HOME` environment variable to point to Java 17 .
+Liberty Tools for Visual Studio Code requires Java 17 or later to ensure that the [Liberty Config Language Server](https://github.com/OpenLiberty/liberty-language-server) and the [Eclipse Language Server for Jakarta EE](https://github.com/eclipse/lsp4jakarta) start properly. A toast message alerts you if the Liberty Language Config server fails to run.
```

Comment:
Use Liberty Config Language Server in last sentence too.

Also, is there a toast message if LSP4Jakarta fails to run?